crates/meilisearch/tests/index/get_index.rs

```diff
@@ -53,19 +53,25 @@ async fn no_index_return_empty_list() {
 
 #[actix_rt::test]
 async fn list_multiple_indexes() {
-    let server = Server::new().await;
-    server.index("test").create(None).await;
-    server.index("test1").create(Some("key")).await;
+    let server = Server::new_shared();
+    let index1 = server.unique_index();
+    let index2 = server.unique_index();
 
-    server.index("test").wait_task(1).await;
+    let (task1, _) = index1.create(None).await;
+    let (_task2, _) = index2.create(Some("key")).await;
+
+    index1.wait_task(task1.uid()).await.succeeded();
```

Comment:
The `wait_task(1)` was referring to the second task :sweat_smile: 
That's also what was causing your task queue corruption I believe (+ the fact we had a bug on main)

```suggestion
    index1.wait_task(task2.uid()).await.succeeded();
```

Reply:
oh..did not realize that it was supposed to be the second task 😅